api_volontaria/apps/volunteer/tests/test_view_events.py

```diff
@@ -260,3 +263,141 @@ def test_list_events(self):
         self.assertEqual(response.status_code, status.HTTP_200_OK)
         self.assertEqual(len(content['results']), 1)
         self.check_attributes(content['results'][0])
+
+    def test_bulk_events_as_users(self):
+        """
+        Ensure we can't bulk add events if we are a simple user.
+        :return:
+        """
+        self.client.force_authenticate(user=self.user)
+
+        response = self.client.post(
+            reverse('event-bulk'),
+            data={'file': BytesIO()},
+            format='multipart'
+        )
+
+        content = json.loads(response.content)
+
+        self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
+        self.assertEqual(
+            content,
+            {'detail': 'You do not have permission to perform this action.'}
+        )
+
+    def test_bulk_events_no_file_is_given(self):
+        """
+        Ensure bad request with explanation is returned if no file
+        is given as input
+        """
+        self.client.force_authenticate(user=self.admin)
+
+        response = self.client.post(
+            reverse('event-bulk'),
+            data={},
+            format='multipart'
+        )
+
+        content = json.loads(response.content)
+
+        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
+        self.assertEqual(
+            content,
+            {'detail': "No file was provided for bulk event creation"}
```

Comment:
Error should reference the field, ex: `{'file': ["No file was provided for bulk event creation"]}`
Note that when we return error on field we keep the messages into arrays to allow multiple errors display for each fields.

This way the frontend will apply the standard error structure and display the error at the best position possible.